doc/releases/changelog-dev.md

```diff
@@ -127,80 +212,158 @@
   ```
   [(#4126)](https://github.com/PennyLaneAI/pennylane/pull/4126)
 
+* The stochastic parameter-shift gradient transform for pulses, `stoch_pulse_grad`, now
+  supports arbitrary Hermitian generating terms in pulse Hamiltonians.
+  [(4132)](https://github.com/PennyLaneAI/pennylane/pull/4132)
+
+<h3>Improvements 🛠</h3>
```

Comment:
Could you please add the following:

```
* Added non-cubic lattice support for all electron resource estimation.
  [(3956)](https://github.com/PennyLaneAI/pennylane/pull/3956)
```

Reply:
Thanks @soranjh, done: 6250756bc8633a388c452e303f2fa19920c5988c.